docs/AboutUs.md

```diff
@@ -4,49 +4,35 @@ We are a team based in the [School of Computing, National University of Singapor
 
 ## Project Team
 
-#### [Damith C. Rajapakse](http://www.comp.nus.edu.sg/~damithch) <br>
-<img src="images/DamithRajapakse.jpg" width="150"><br>
-**Role**: Project Advisor
+#### [Desmond Ang](https://github.com/KnewYouWereTrouble)
+<img src="images/Desmond.jpg" width="150"><br>
+**Role**: Team Lead, Git Expert <br>
+**Responsibilities**: Deliverables and Deadlines
```

Comment:
in the context of this project, responsibility means, who is in charge of which component?
who is doing UI, who is doing logic etc.